llettuce-docs/_build/html/_sources/autoapi/prompt/index.rst.txt

```diff
@@ -1,24 +1,26 @@
-prompt
-======
+:py:mod:`prompt`
+================
 
 .. py:module:: prompt
 
 
+Module Contents
+---------------
+
 Classes
--------
+~~~~~~~
 
 .. autoapisummary::
 
    prompt.Prompts
 
 
-Module Contents
----------------
+
 
 .. py:class:: Prompts(model_name: str, prompt_type: str | None = 'simple')
 
-   This class is used to generate prompts for the models.
 
+   This class is used to generate prompts for the models.
```

Comment:
what is the objective of the prompts and how are they generated? worth it adding a few more words here?